bin/oref0-pump-loop.sh

```diff
@@ -149,19 +149,57 @@ function overtemp {
 }
 
 function smb_reservoir_before {
+    
     # Refresh reservoir.json and pumphistory.json
     try_fail refresh_pumphistory_and_meal
     try_fail cp monitor/reservoir.json monitor/lastreservoir.json
-    try_fail timerun openaps report invoke monitor/clock.json monitor/clock-zoned.json 2>&3 >&4 | tail -1
+        CURRENT_DTG=$(date +%s)
+    try_fail timerun openaps report invoke monitor/clock.json monitor/clock-zone                                                                                                                                                                              d.json 2>&3 >&4 | tail -1
```

Comment:
This and a bunch of other lines are corrupted with a bunch of extra whitespace.  Can you fix and open a new PR?